app/src/main/res/layout/activity_information.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
There are a few other suggestions by Android Studio. Switch to Code View and hover on the lines and use `Alt + Enter` to fix these issues.